java/api/src/main/java/org/ray/api/UniqueID.java

```diff
@@ -12,6 +12,8 @@
 public class UniqueID implements Serializable {
 
   public static final int LENGTH = 20;
+  public static final int OBJECT_INDEX_POS = 0;
+  public static final int OBJECT_INDEX_LENGTH = 4;
```

Comment:
These 2 constants should be in `UniqueIdHelper`

Reply:
All static methods and static members in `UniqueIDHelper` can be moved into `UniqueID`  because `UniqueIdHlper` class is useless in my view.